stable/aws-for-fluent-bit/templates/configmap.yaml

```diff
@@ -127,6 +127,29 @@ data:
         {{- end }}
 {{- end }}
 
+{{- if .Values.elasticsearch.enabled }}
+    [OUTPUT]
+        Name            es
+        Match           {{ .Values.elasticsearch.match }}
+        AWS_Region      {{ .Values.elasticsearch.region }}
+        AWS_Auth        {{ .Values.elasticsearch.aws_auth }}
```

Comment:
Did you want to include either `AWS_Role_ARN` or `AWS_External_ID`?

Reply:
I opted to specify the Arn using Iam Role for Service Account, so didn't think this makes sense to specify it here, but I can added it if you think otherwise